qwed_sdk/cli.py

```
@@ -2082,5 +2082,83 @@ def import_provider(url: str):
+    from qwed_new.core.verification_context import (
+        VerificationContextValidationError,
+        validate_document,
+    )
+
+    with open(path, encoding="utf-8") as handle:
+        document = json.load(handle)
+    try:
+        validate_document(document)
+    except VerificationContextValidationError as exc:
+        click.echo(json.dumps({"valid": False, "error": str(exc)}, indent=2))
+        raise SystemExit(1)
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)